public/app/plugins/panel/canvas/editor/options.ts

```diff
@@ -120,4 +123,20 @@ export const optionBuilder: OptionSuppliers = {
       },
     });
   },
+
+  addType: (builder, context) => {
+    const category = ['Type'];
```

Comment:
Perhaps instead of "Type", we call it something more explicit, like "Line Style". I believe we may want to use "Type" when referring to connections in the future for different default types of connections: curved, orthogonal, etc.